src/data.ts

```diff
@@ -0,0 +1,35 @@
+require('dotenv').config();
+
+const data ={
+    "contactInfo": {
+        "fullName": "Isaias Samuel Puga Solares",
+        "emailAddress": "puga.isaias@gmail.com"
+    },
+    "github": {
+        "profileURL": "https://github.com/pugaIsaias",
+        "username": "pugaIsaias"
+    },
+    "credentials": {
+        "password": process.env.API_KEY
```

Comment:
Este sería un nombre más adecuado: 

```suggestion
        "password": process.env.PASSWORD
```

Reply:
Muy bien ya corregí el nombre de la variable.